File: Lua/bee 1019.lua
local n = tonumber(io.read("*n"))

local horas = math.floor(n/3600)
local resto = n % 3600
local minutos = math.floor(resto/60)
resto = resto % 60
local segundos = resto

print(string.format("%d:%d:%d", horas, minutos, segundos))

File: Rust/bee_1019.rs
use std::io::{self, Read};

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();

    let tempo_segundos: i32 = input.trim().parse().unwrap();

    let horas = tempo_segundos / 3600;
    let mut resto = tempo_segundos % 3600;

    let minutos = resto / 60;
    resto %= 60;

    let segundos = resto;

    println!("{}:{}:{}", horas, minutos, segundos);
}
